src/Admin/AbstractAdmin.php

```diff
@@ -3029,7 +3029,7 @@ protected function buildForm()
 
             $value = $propertyAccessor->getValue($object, $propertyPath);
 
-            if (\is_array($value) || ($value instanceof \Traversable && $value instanceof \ArrayAccess)) {
+            if (is_iterable($value) || $value instanceof \ArrayAccess) {
```

Comment:
In fact, I think this check must be replaced. A value implementing `\Traversable` doesn't guarantee that it can be handled as an array.
```suggestion
            if (\is_array($value) || $value instanceof \ArrayAccess) {
```

Reply:
It would be great if php had a way to check this in one go.

Reply:
Yes, I agree... maybe we should propose a RFC for this.